src/interface/web/.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn run lint-staged
+yarn test
```

Comment:
Will this automatically work in our pre-commit hook? If so, neat.

Reply:
Note: it will not. Would have to manually run the `yarn` command.

Reply:
Yeah, `yarn lint --fix` or `next lint --fix` should apply prettier formatting. Don't want to do it right now as we're building a bunch of things in parallel, so it'll create unnecessary conflicts. But we should run it before we merge to master and enable it to run as a pre-commit hook from then on